packages/functional-tests/tests/settings/password.spec.ts

```diff
@@ -6,109 +6,141 @@ import { test, expect } from '../../lib/fixtures/standard';
 import { EmailHeader, EmailType } from '../../lib/email';
 
 test.describe('severity-1 #smoke', () => {
-  // https://testrail.stage.mozaws.net/index.php?/cases/view/1293389
-  test('forgot password #1293389', async ({
+  test('forgot password - react', async ({
```

Comment:
I'm not sure why these reset password tests are in the settings section. Perhaps they initially started from the changePassword flow and followed the `Forgot password?` link? 

First one should probably be moved to `react-conversion/resetPassword` (this is reset password _same_ tab scenario, existing test in the resetPassword folder is reset password _different_ tab scenario)

Second one to `reset-password/resetPassword`

Third to `react-conversion/recoveryKey`

Fourth perhaps `reset-password/resetPassword`?

Reply:
Yes to 1-3, opted to put the fourth in `settings/recoveryKey` cause it seems more symmetrical with `react-conversion/recoveryKey` 🤷🏻‍♀️  (see https://github.com/mozilla/fxa/pull/16671/commits/5fa5f3fc13dbdc8eec20e533fda9556f385dec18)